multiworld/envs/mujoco/sawyer_xyz/sawyer_push_multiobj.py

```diff
@@ -334,6 +345,27 @@ def reset_mocap2body_xpos(self):
             np.array([self.data.body_xquat[self.endeff_id]]),
         )
 
+    def set_initial_object_positions(self):
+        if self.fixed_start: # set object to middle of workspace always
+            for i in range(self.num_objects):
```

Comment:
Will this break if there's multiple objects?

Reply:
Kind of, it sets them all to be in the same spot. I'll add an assert